apps/src/templates/Buttons.jsx

```diff
@@ -0,0 +1,92 @@
+var msg = require('../locale');
```

Comment:
Sorry, wasnt clear. For Buttons.jsx. I imagine we will have other components that are buttons that it would be nice to differentiate these buttons from.